.github/workflows/fuzzers.yml

```diff
@@ -107,6 +107,15 @@ jobs:
           - afl_maxmap
           - introspector_driven_focus
           - libfuzzer_focus_idx0
+          - libfuzzer_focus_idx1
```

Comment:
We don't want this. It will clog up CI.
actually @Navidem can we get rid of this or will you need more experiments?